include/envoy/grpc/context.h

```diff
@@ -60,6 +60,24 @@ class Context {
    */
   virtual void chargeStat(const Upstream::ClusterInfo& cluster, const RequestNames& request_names,
                           bool success) PURE;
+
+  /**
+   * Charge a request message stat to a cluster/service/method.
+   * @param cluster supplies the target cluster.
+   * @param request_names supplies the request names.
+   * @param amount supplies the number of the request messages.
+   */
+  virtual void chargeRequestStat(const Upstream::ClusterInfo& cluster,
```

Comment:
nit: RequestMessage and ResponseMessage below.

Reply:
Done.